spark/dl/src/main/scala/com/intel/analytics/bigdl/python/api/PythonBigDL.scala

```diff
@@ -99,6 +101,23 @@ class PythonBigDL[T: ClassTag](implicit ev: TensorNumeric[T]) extends Serializab
       cls.getSimpleName)
   }
 
+  def toTensor(jTensor: JTensor): Tensor[T] = {
+    Tensor(jTensor.storage.asScala.toArray.asInstanceOf[Array[T]],
+      jTensor.shape.asScala.toArray)
+  }
+
+  def toJTensor(tensor: Tensor[T]): JTensor = {
+    // TODO: we should clone here, in case the underlying storage large than the tensor size.
+    val cloneTensor = tensor
```

Comment:
Don't use clone - you need to make sure it's continuous. use resizeAs() and copy() instead.